Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
```

Comment:
use slim version 
please read [checklist.md](https://github.com/mate-academy/py-docker-weather-api/blob/master/checklist.md)

Reply:
Still not fixed
```suggestion
FROM python:3.10.8-slim
```